Common/config/language/partials/en_GB/markup-cookie-detail-last-updated.phtml

```diff
@@ -1,3 +1,3 @@
-<div class="app-c-published-dates govuk-!-margin-top-7" lang="en">
-    Last updated 31 January 2020
-</div>
+<p class="govuk-body govuk-!-margin-top-7" lang="en">
+  Last updated 8 April 2025
```

Comment:
Only the one small thing here - the page has been updated in terms of markup, but the cookie policy itself hasn't been. So this date still needs to be 31 January 2020. Otherwise all LGTM.
```suggestion
  Last updated 31 January 2020
```